packages/block-library/src/navigation/edit/index.js

```diff
@@ -515,6 +521,11 @@ function Navigation( {
 											! overlayMenuPreview
 										);
 									} }
+									aria-label={ __(
+										'Toggle overlay menu controls'
```

Comment:
Reason: Buttons already let screen reader users know that it is a toggle, `aria-expanded` also gives us clues. Just need to communicate what the button does in text and the ARIA attributes tell us the rest.

```suggestion
										'Overlay menu controls'
```